contrib/format-httpd/README.md

```diff
@@ -0,0 +1,32 @@
+# Web Server Log Format Plugin (HTTPD)
+This plugin enables Drill to read and query httpd (Apache Web Server) and nginx logs natively. This plugin uses the work by [Niels Basjes](https://github.com/nielsbasjes) which is available here: https://github.com/nielsbasjes/logparser.
```

Comment:
It only does Access logs (no error logs or any other kind of logs)

Reply:
Fixed